### ISSUE TITLE
[Feature]: Share whole library with other user

### ISSUE THREAD

**Issue by anpavlov** [immich-app/immich#637]
### Feature detail

It would be great to see a possibility to share whole library with another user, so that they could view my all photos without editing (uploading into my library).

### Platform

Server

**Comment by llexaxell** [comment]
Is it related to these issues ?
- https://github.com/immich-app/immich/issues/628
- https://github.com/immich-app/immich/issues/265

**Comment by anpavlov** [comment]
I don't think so. My suggestion is about sharing within single immich instance. My main reason is ability to share library with my wife, so that she can view my photo library anytime, while i can view her library.

**Comment by sbkg0002** [comment]
> I don't think so. My suggestion is about sharing within single immich instance. My main reason is ability to share library with my wife, so that she can view my photo library anytime, while i can view her library.

For me this is a feature to be able to replace Gphotos.
